core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -510,7 +510,6 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with TimeLi
     assert(initialMapStatus1.map{_.mapId}.toSet === Set(5, 6, 7))
 
     val initialMapStatus2 = mapOutputTracker.shuffleStatuses(secondShuffleId).mapStatuses
-    //  val initialMapStatus1 = mapOutputTracker.mapStatuses.get(0).get
```

Comment:
Let's revert this change. This kind clean up is good in general of course, but this makes the backport difficult.